src/lib/core/WeaveTLV.h

```diff
@@ -141,6 +141,7 @@ friend class TLVUpdater;
     uint32_t GetRemainingLength(void) const { return mMaxLen - mLenRead; }
 
     const uint8_t *GetReadPoint(void) const { return mReadPoint; }
+    uintptr_t GetBufHandle(void) const { return mBufHandle; }
```

Comment:
Out of curiosity, is there a use case for this?

Reply:
It's something I'm not particularly happy with: I needed to create a brand new circular buffer and associated reader within the storage used by an existing reader.